wordpress-mobile/gutenberg-mobile
# Moving RNAztecView.podspec to root dir.

This PR solves an issue when trying to load RNAztecView as a subtree of `gutenberg-mobile` using cocoapods on WPiOS: https://github.com/wordpress-mobile/WordPress-iOS/pull/10864

To test:
- Check out https://github.com/wordpress-mobile/WordPress-iOS/pull/10864 on WPiOS
- Delete `Pods` folder from the WPiOS project folder
- Run `rake dependencies`
- Check that all the pods are installed correctly (specially `rn-aztec` from `gutenberg-mobile`)
- Run the app and test that gutenberg runs properly.
